README.md

```diff
@@ -27,7 +33,8 @@ Usage:
 - For instance, typing `/giphy Hello, strange world of giphy!` will display,
 <img src="https://user-images.githubusercontent.com/1187448/63696085-cf806780-c7ce-11e9-9c77-a4fa8c693bf0.png" width="500"/>
 
-This is presented only to the user who typed the message and is not posted to the channel, yet.
+This is presented only to the user who typed the message and is not posted to
```

Comment:
Sentence restructure.  

This is presented only to the user who typed the message but is not yet posted to the channel.